tests/acceptance/features/apiGraph/getUser.feature

```diff
@@ -0,0 +1,93 @@
+@api @skipOnOcV10
+Feature: get users
+  As an admin
+  I want to be able to retrieve user information
+  So that I can see the information
+
+  Background:
+    Given user "Alice" has been created with default attributes and without skeleton files
+    And the administrator has given "Alice" the role "Admin" using the settings api
+
+
+  Scenario: admin user gets the information of a user
+    Given user "Brian" has been created with default attributes and without skeleton files
```

Comment:
The user `Brian` is created for every scenario, so how about adding this to `Background`?